tattoo_shop/demo/website_views.xml

```diff
@@ -0,0 +1,929 @@
+                          <h4 class="h5-fs">Iris Joe</h4>
+                          <p class="text-muted mb-3">Chief Marketing Officer</p>
+                          <p>John, with his years of experience in the industry, ensures that our services are of the highest quality. He is dedicated to providing exceptional customer service and taking our business to new heights.</p>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </section>
+            </div>
+          </div>
+        </t>
+      </t>
+    </field>
+    <field name="key">tattoo_shop.website.about-us</field>
+    <field name="name">About Us</field>
+    <field name="type">qweb</field>
+    <field name="website_id" ref="website.default_website"/>
+  </record>
+  <record id="ir_ui_view_2945" model="ir.ui.view">
+    <field name="arch" type="xml">
+      <t name="Appointment Types" track="1" t-name="website_appointment.appointments_cards_layout">
+        <t t-call="website.layout">
+          <div id="wrap" class="o_appointment_index">
+            <div class="oe_structure"/>
+            <t t-call="website_appointment.website_calendar_index"/>
+            <t t-call="website_appointment.website_calendar_index_topbar"/>
+            <div class="o_appointment_choice o_appointment_appointments_list">
+              <div class="container">
+                <div id="o_appointment_index_main_col" class="col-md px-0 o_wappointment_appointment_list_columns">
+                  <div class="row">
+                    <t t-call="website_appointment.appointments_cards"/>
+                  </div>
+                </div>
+              </div>
+              <div class="oe_structure oe_empty">
+                <section class="s_title pb40 o_colored_level pt24" data-vcss="001" data-snippet="s_title" data-name="Title">
+                  <div class="container s_allow_columns">
+                    <h2 class="display-3-fs" style="text-align: center;">Custom Tattoo?&amp;nbsp;</h2>
+                    <p style="text-align: center;">Get in touch with us to discuss your project.&amp;nbsp;</p>
+                    <p style="text-align: center;">
+                      <a href="/contactus" class="btn btn-primary btn-lg">Discuss your project</a>
+                      <br/>
+                    </p>
+                  </div>
+                </section>
+              </div>
+            </div>
+          </div>
+        </t>
+      </t>
+    </field>
+    <field name="key">tattoo_shop.website_appointment.appointments_cards_layout</field>
+    <field name="name">Appointment Types</field>
+    <field name="type">qweb</field>
+    <field name="website_id" ref="website.default_website"/>
+  </record>
+  <record id="ir_ui_view_2939" model="ir.ui.view">
+    <field name="arch" type="xml">
+      <t name="Appointment: Appointment Info" t-name="appointment.appointment_info">
```

Comment:
I don't see what is the difference between this view and the standard `appointment.appointment_info`